tools/font-edit/twin-fedit.c

```diff
@@ -22,57 +22,45 @@
 
 #include "twin-fedit.h"
 
-static Display *dpy;
-static Window win;
-static Visual *visual;
-static int depth;
+static SDL_Window *window;
+static cairo_t *cr;
+static cairo_surface_t *surface;
+
 static int width = 512;
 static int height = 512;
 static double scale = 8;
-static cairo_t *cr;
-static cairo_surface_t *surface;
 static int offset;
 
 static int offsets[1024];
 
+/* Control the life of the widown,
+ * if the value is 0, the windown lives,
+ * otherwise the windown quit.
+ */
+static int exit_window = 0;
```

Comment:
Use `bool` when possible.

Reply:
Re solve in 7231d62d0106fb56594ddd7785efcc9ab9779b04.